website/snippets/_config-dbt-version-check.md

```diff
@@ -0,0 +1,19 @@
+
+Starting in 2024, when you select **Keep on latest version** in dbt Cloud, dbt will ignore the `require-dbt-version` config. Refer to [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) (available as a beta) for more details. 
+
+dbt Labs is committed to zero breaking changes for code in dbt projects, with ongoing releases to dbt Cloud and new versions of dbt Core. We also recommend these best practices:
+
```

Comment:
I wonder if instead of having these two bullets of detail here, they should live in the pages I linked to below and we can summarize here and send them to that page for details. Just a thought.